caluma/caluma_core/tests/test_mutation.py

```diff
@@ -77,7 +77,7 @@ class Meta:
             serializer_class = MySerializer
 
     result = NoModelMutation.mutate_and_get_payload(None, info, name="test")
-    assert type(result) == NoModelMutation
+    assert type(result) is NoModelMutation
```

Comment:
Same here again, unless you specifically don't want to allow any subtypes you can use `isinstance()` instead